RELEASE.md

```diff
@@ -0,0 +1,14 @@
+Release type: refactor
```

Comment:
```suggestion
Release type: minor
```

Reply:
oh that makes sense didn't realize it was for semver